src/js/jsx/sections/style/UnsupportedEffectList.jsx

```diff
@@ -161,6 +163,8 @@ define(function (require, exports, module) {
                 );
             }
             
+            headlights.logEvent("effect", "create", "unsupported");
```

Comment:
I think we need to get rid of all logging calls from `render` methods, even if that means we can't log some things.